source/tests/system/nidaqmx_driver_api_tests.cpp

```diff
@@ -267,8 +291,8 @@ class NiDAQmxDriverApiTests : public Test {
   template <typename TResponse>
   void EXPECT_SUCCESS(const ::grpc::Status& status, const TResponse& response)
   {
-    EXPECT_TRUE(status.ok());
     EXPECT_EQ(DAQmxSuccess, response.status());
+    EXPECT_EQ(::grpc::Status::OK.error_code(), status.error_code());
```

Comment:
Just curious, what does this change do? Does it give better output on failure or something?

Reply:
Yeah, the original assert will just say "expected false, actual true". This will tell you the error code you got so it's easier to guess what happened.